.github/workflows/build_pr_documentation.yml

```
@@ -17,4 +17,4 @@ jobs:
       package_name: audio-course
       path_to_docs: audio-transformers-course/chapters/
       additional_args: --not_python_module
-      languages: en
\ No newline at end of file
+      languages: en, ben
```

```suggestion
      languages: en ben
```

as [here](https://github.com/huggingface/course/blob/main/.github/workflows/build_pr_documentation.yml#L19), we just list them without comma 😊